Bio/MaxEntropy.py

```diff
@@ -25,6 +25,58 @@ class MaxEntropy(object):
     alphas       List of the weights for each feature.
     feature_fns  List of the feature functions.
 
+    Car data from example Naive Bayes Classifier example by Eric Meisner November 22, 2003
+    http://www.inf.u-szeged.hu/~ormandi/teaching
+    >>> from Bio.MaxEntropy import train, classify
+    >>> xcar = [
+    ...     ['Red', 'Sports', 'Domestic'],
+    ...     ['Red', 'Sports', 'Domestic'],
+    ...     ['Red', 'Sports', 'Domestic'],
+    ...     ['Yellow', 'Sports', 'Domestic'],
+    ...     ['Yellow', 'Sports', 'Imported'],
+    ...     ['Yellow', 'SUV', 'Imported'],
+    ...     ['Yellow', 'SUV', 'Imported'],
+    ...     ['Yellow', 'SUV', 'Domestic'],
+    ...     ['Red', 'SUV', 'Imported'],
+    ...     ['Red', 'Sports', 'Imported']]
+    >>> ycar = ['Yes','No','Yes','No','Yes','No','Yes','No','No','Yes']
+
+    Requires some rules or features
+    >>> def udf1(ts, cl):
+    ...     if ts[0] == 'Red':
+    ...         return 0
+    ...     else:
+    ...         return 1
```

Comment:
Oh yeah - shows how used I am to Python's ``False`` and ``True`` compared to C and very early Python's ``0`` and ``1`` (or rather, any non-zero value) style.